composer.json

```diff
@@ -2,9 +2,14 @@
 	"scripts": {
 		"cs:check": "php-cs-fixer fix --dry-run --diff",
 		"cs:fix": "php-cs-fixer fix",
-		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -not -path './tests/integration/vendor/*' -print0 | xargs -0 -n1 php -l"
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -not -path './tests/integration/vendor/*' -print0 | xargs -0 -n1 php -l",
+		"psalm": "psalm",
+		"psalm:fix": "psalm --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType"
 	},
+	"minimum-stability": "dev",
 	"require-dev": {
-		"nextcloud/coding-standard": "^0.3.0"
+		"nextcloud/coding-standard": "^0.3.0",
+		"vimeo/psalm": "^3.16",
+		"christophwurst/nextcloud": "dev-master"
```

Comment:
you can drop it here and only install on CI

Reply:
but then one can not run it locally?
Or is it recomment to install this and psalm locally manually?

Reply:
Yeah, right. You can leave it then. Or install the OCP package only on demand when you want to psalm locally. I'd keep psalm definitely in composer.json as that locks the version.